configs/proactive-scaler/base/deployments.yaml

```diff
@@ -0,0 +1,96 @@
+---
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: m6a-2xlarge
+  namespace: proactive-scaler
+spec:
+  replicas: 1
+  selector:
+    matchLabels:
+      run: m6a-2xlarge
+  template:
+    metadata:
+      labels:
+        run: m6a-2xlarge
+    spec:
+      nodeSelector:
+        konflux-ci.dev/workload: konflux-tenants
+        node.kubernetes.io/instance-type: m6a.2xlarge
+      tolerations:
+        - key: konflux-ci.dev/workload
+          operator: "Equal"
+          value: "konflux-tenants"
+          effect: "NoSchedule"
+      priorityClassName: pause-pods
+      containers:
+        - name: reserve-resources
+          image: registry.k8s.io/pause
+          resources:
+            requests:
+              memory: "22Gi"
+              cpu: "5"
+---
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: m6a-4xlarge
+  namespace: proactive-scaler
+spec:
+  replicas: 1
+  selector:
+    matchLabels:
+      run: m6a-4xlarge
+  template:
+    metadata:
+      labels:
+        run: m6a-4xlarge
+    spec:
+      nodeSelector:
+        konflux-ci.dev/workload: konflux-tenants
+        node.kubernetes.io/instance-type: m6a.4xlarge
+      tolerations:
+        - key: konflux-ci.dev/workload
+          operator: "Equal"
+          value: "konflux-tenants"
+          effect: "NoSchedule"
+      priorityClassName: pause-pods
+      containers:
+        - name: reserve-resources
+          image: registry.k8s.io/pause
+          resources:
+            requests:
+              memory: "45Gi"
+              cpu: "10"
+---
+apiVersion: apps/v1
+kind: Deployment
```

Comment:
Hello @hugares ! I thought we will just create CronJob that will trigger a Job with 1 second pause every minute or so. I do not understand this too much, but this Deployment will just create pods that will be handing there forever, so they will not make the resources available. Or does it work differently?

Reply:
This change is creating 2 priority classes, default one(for all pods) will have higher priority that the dummy deployments. Dummy deployments get schedule on node A, as soon as a pod need to be scheduled, that does not fit on any other node than node A, then dummy pods get evicted(takes less than a second) and pod get scheduled on node A. The dummy pod, will be pending and cause the autoscaler to add another node.

Reply:
I know we talked about cron but this solution is more elegant and has the advantage to not having to wait until cron runs again to cause auto scaling